### ISSUE TITLE
Run a job every N days/weeks/months/years at a specific time

### ISSUE THREAD

**Issue by renom** [distribworks/dkron#1048]
For example:
Every 3 days at 15:00:00.

**Comment by vcastellm** [comment]
You can use something like https://crontab.guru/#0_15_*/3_*_*

**Comment by renom** [comment]
It's not exactly what I mean.
The `*/3` notation in your example implies triggering on `1 4 7 10 ...` within every month.
My suggestion is allowing to combine `@every` and `@at` so that the first trigger of `@every` will occur at the provided date.

**Comment by yvanoers** [comment]
This is a similar to what is discussed in #823.

Because #823 was initially a misunderstanding of what Dkron uses as expressions, and this issue is an actual feature request I will quote the last most relevant part of that discussion here and then close that issue.

@vcastellm said:
> I didn't know about RRULE, I think it should be possible to implement a schedule parser to handle RRULE expressions, the other alternative I'm inclined for in terms of expression expressivity is using https://en.wikipedia.org/wiki/ISO_8601 repeating interval notation.

**Comment by renom** [comment]
Implementing ISO 8601 repeating intervals really makes sense. The question is should it be implemented in dkron or in [robfig/cron](https://github.com/robfig/cron)?
By the way, [the implementation in ajvb/kala](https://github.com/ajvb/kala/blob/master/job/job.go#L257) could help here.

**Comment by vcastellm** [comment]
Yeah I had the kala reference saved for the future, but I don't know how or when this is going to be implemented.

**Comment by IgnisDa** [comment]
@yvanoers Any updates on RRULES or repeating intervals?
